contracts/truefi2/PoolFactory.sol

```diff
@@ -132,16 +160,49 @@ contract PoolFactory is IPoolFactory, UpgradeableClaimable {
         emit PoolCreated(token, address(proxy));
     }
 
+    /**
+     * @dev Create a new private pool behind proxy. Update new pool's implementation.
+     * Transfer ownership of created pool to Factory owner.
+     * @param token Address of token which the pool will correspond to.
+     */
+    function createSingleBorrowerPool(
+        address token,
+        string memory borrowerName,
+        string memory borrowerSymbol
+    ) external onlyAllowedTokens(token) onlyWhitelistedBorrowers(msg.sender) {
```

Comment:
I'm not sure if we want whitelisted borrowers to be able to make a pool for every token. I will get clarification on this from the team. @yuchenlintt